core/spec/models/spree/store_credit_spec.rb

```diff
@@ -868,6 +957,25 @@
         expect { subject }.to change { store_credit.store_credit_events.where(action: Spree::StoreCredit::INVALIDATE_ACTION).count }.from(0).to(1)
       end
 
+      it "adds an entry to the ledger" do
+        expect { subject }.to change { Spree::StoreCreditLedgerEntry.count }.by(1)
```

Comment:
Rather than checking `Spree::StoreCreditLedgerEntry.count` I think it makes more sense to check: `store_credit.store_credit_ledger_entries.count`